docker-compose.yaml

```diff
@@ -16,6 +16,7 @@ services:
       - LOCALSTACK_AUTH_TOKEN=${LOCALSTACK_AUTH_TOKEN}
       - PERSISTENCE=1
       - SNAPSHOT_SAVE_STRATEGY=ON_REQUEST
+      - EXTRA_CORS_ALLOWED_ORIGINS=*
```

Comment:
This is required as currently, we return a URL to the calling client which points to the local stack resource.